Modules/_ssl/cert.c

```diff
@@ -153,13 +153,13 @@ _x509name_print(_sslmodulestate *state, X509_NAME *name, int indent, unsigned lo
  * PySSLCertificate_Type
  */
 
-#define _PySSLCertificate_CAST(op)  ((PySSLCertificate *)(op))
```

Comment:
I don't thinkt that this change is useful.

Reply:
Note: it's something I've changed last month, and I thought I fixed all the UBSan issues in the SSL module but this wasn't the case. I can revert it if you don't want this additional diff.